py/torch_tensorrt/fx/fx2trt.py

```diff
@@ -259,6 +302,9 @@ def run(
         _LOGGER.info(
             f"Build TRT engine elapsed time: {datetime.now() - build_engine_start_time}"
         )
+        _LOGGER.info(
+            f"TRT Engine uses: {engine.device_memory_size} Memory"
```

Comment:
`f"TRT Engine uses: {engine.device_memory_size} Memory"` -> `f"TRT Engine uses {engine.device_memory_size} bytes of memory"`

Reply:
Fixed it.